governance-exercises/VoteWizard_poll_exercise_01/Adjust Spork D3M Parameters and DAI Interest Rate Strategy - July 24, 2023.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Adjust Spork D3M Parameters and DAI Interest Rate Strategy - July 24, 2023
+summary: Signal your support or opposition to adjusting the Spork DAI Direct Deposit Module (D3M) parameters.
```

Comment:
The template summaries are fairly bad. Feel free to put something more useful if you want to.